modules/ztp-image-based-upgrade-creating-backup-resources.adoc

```diff
@@ -150,14 +158,64 @@ metadata:
 [...]
 ----
 
-.. Apply the `ConfigMap`:
+.. If you are using {rh-rhacm}, apply the `ConfigMap`:
++
+[source,terminal]
+----
+$ oc apply -f oadp-cm-example.yaml
+----
+
+.. If you are using GitOps ZTP, push the generated `ConfigMap` object to your git repository.
+
+. (Optional) To keep your custom catalog sources after the upgrade, generate another `ConfigMap` object for your catalog sources and add them to the `spec.extraManifest` in the `ImageBasedUpgrade` CR. For more information about catalog sources, see xref:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html-single/operators/index#olm-catalogsource_olm-understanding-olm[Catalog source].
+
+.. Create the `CatalogSource` CR.
++
+--
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  annotations:
+    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+  name: example-catalogsources
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  displayName: disconnected-redhat-operators
+  image: quay.io/example-org/example-catalog:v1
+----
+--
+
+.. Create the `ConfigMap` object:
++
+--
+[source,terminal]
+----
+$ oc create configmap example-extra-manifests-cm --from-file=example-catalogsources.yaml=<path_to_catalogsource_cr> -n openshift-lifecycle-agent
+----
+
+
+.Example output
```

Comment:
That's not the output. The expected output is:
`
configmap/example-extra-manifests-cm created
`